aea/cli/ipfs_hash.py

```diff
@@ -245,86 +225,10 @@ def update_hashes(
                 if vendor is not None and package_id.author != vendor:
                     continue
                 package_hashes[key] = package_hash
-
-        to_csv(package_hashes, packages_dir / HASHES_FILE)
         click.echo("Done!")
-
-    except Exception:  # pylint: disable=broad-except
-        traceback.print_exc()
-        return_code = 1
-
-    return return_code
-
-
-def check_same_ipfs_hash(
```

Comment:
Where is this happening now?

Reply:
https://github.com/valory-xyz/open-aea/blob/feat/package-sync/aea/cli/packages.py#L295-L332